lib/sse.js

```diff
@@ -126,7 +149,19 @@ SSE.prototype.requestValidationMiddleware = function(req, res, next) {
   next();
 };
 
-SSE.prototype.handler = function(req, res) {
+SSE.prototype.handler = function (req, res) {
```

Comment:
We have replaced the handler function. In case we go ahead with this approach, we should use the consider integrate the kafka `consumer` on it.